pyproject.toml

```diff
@@ -14,7 +14,7 @@ classifiers = [
 ]
 dependencies = [
     "astropy>=5.0.4",
-    "numpy",
+    "numpy>=2.0",
```

Comment:
I suggest "astropy>=6.0" as this is the minimum version the astropy team supports.

Reply:
added.